participating.md

```diff
@@ -1,5 +1,70 @@
+    - **Feasibility**: How feasible is your idea in terms of technical, financial, and time resources?
+    - **Scalability**: How scalable is your idea in terms of potential impact and applicability to different contexts, industries and domains?
+    - **Replicability**: How replicable is your idea in terms of sharing best practices and learnings with other software practitioners and stakeholders?
+- **Idea validation**: To validate your idea, you should use the IF tools and resources to estimate the environmental impact of your software solution and compare it with the baseline or alternative solutions. You should also seek feedback from your peers, mentors, and experts on the Carbon Hack 24 platform and community.
+
+
+## 💚 What prize category should we go for?
+#### **Prize categories**: _There are four prize categories for Carbon Hack 24:_
+- 💧 Beyond Carbon
+Any submission that demonstrates the application of Impact Framework beyond carbon. (e.g. water, waste, air quality).
+
+- 📦 Best Model
+A model plugin that best supports and enhances the Impact Framework model plugin ecosystem.
+
+- 📝 Best Content
+Best piece of content published about Impact Framework, an article, tutorial, video or how to guide.
+
+- ✨ Best Contribution to the Framework
+Best contribution to the Impact Framework tool itself. Adding a new feature or enhancing an existing one.
+
+#### We are also keen to broaden the opportunities for those who are studying:
+- 🎓 Best Undergraduate
+Specific prize for Undergraduate Students, drawn from across the main categories.
+
+
+#### **Prize selection**: To select the prize category that best suits your idea, you should consider the following questions:
+- **What is the main goal of your idea?**: 
+- **What is the main strength of your idea?**:
+- **What is the main challenge of your idea?**: Is it the relevance, the feasibility, the scalability, or the replicability of your idea?
+- **What is the main audience of your idea?**: Is it software practitioners, software users, software stakeholders, or the general public?
+ 
+
+## 💚 I need help on my project
+- **Project support**: Carbon Hack provides various forms of support for your project, such as¹:
+    - **Access to experts**: Our Impact Framework team are here to help! You can request support through the Carbon Hack Discussions forum.
+    - **Resources**: You can access resources such as tutorials, guides, webinars, and case studies on how to use IF and other green software tools and practices. You can find these resources on the IF website.
+    - **Community & Orihect Feedback**: You can connect with other participants, experts, and the organizers of Carbon Hack 24 through the Carbon Hack discussions forum on GitHub, and the social media channels of the Green Software Foundation. You can also join the events and activities organized by Carbon Hack 24, such as ideation sessions, workshops, and networking sessions.
+
+
+## 💚 Registering & Submitting Ideas
```

Comment:
Turn this into a step by step process people can follow, click link, fill in details, press button. Currently there isn't much point reading the text below since you can't actually follow the instructions to register. If there are links that don't exist yet make them TBD.

The details are in this ticket: https://github.com/Green-Software-Foundation/carbonhack-event/issues/170

1. You create an issue with details of your project team.
2. You recruit a team if you want.
3. When you are ready you register the project team with a form thats TBD
4. You get added to a mail thread with all your team mates and us. You're officially taking part.
5. When ready you submit your solution with a form thats TBD.